docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -353,6 +353,10 @@ children:
     url: Extension-life-cycle.html
     output: 'web, pdf'
 
+  - title: 'Extension point and extensions'
```

Comment:
Since "Extending request body parsing" and "Extension life cycle" is building on top of "Extension point", I feel the sidebar should list "Extension point" first, before the other two pages.